src/rest-api/stats/class-endpoint-posts.php

```diff
@@ -214,6 +215,23 @@ public function sanitize_urls( array $urls ): array {
 		return array_map( 'sanitize_url', $urls );
 	}
 
+	/**
+	 * Validates if the provided array is a list of URLs.
+	 *
+	 * @since 3.18.0
+	 *
+	 * @param array<string> $urls The array to validate.
+	 * @return true|WP_Error
+	 */
+	public function validate_urls( array $urls ) {
+		foreach ( $urls as $url ) {
+			if ( false === filter_var( $url, FILTER_VALIDATE_URL ) ) {
+				return new WP_Error( 'invalid_param', __( 'The parameter must be a list of URLs.', 'wp-parsely' ) );
+			}
+		}
+
+		return true;
+	}
```

Comment:
Let's add an empty line here.

```suggestion
	}

```